src/test/app/Check_test.cpp

```diff
@@ -608,7 +631,7 @@ class Check_test : public beast::unit_test::suite
             // because one unit of alice's reserve is released when the
             // check is consumed.
             env (check::cash (bob, chkId, check::DeliverMin (checkAmount)));
-            env.close();
+            verifyDeliveredAmount (env, drops(checkAmount.mantissa()));
```

Comment:
This feels a bit like splitting hairs, but does `env.tx()` work if you call `env.close()` first? It would be cool to verify that the delivered amount field also gets populated correctly after a call to `env.close()`. Even if it doesn't, you could maybe do a two-part call, where you save the txid before calling `env.close()` then do the same checks.

Reply:
:man_facepalming: 

I completely missed the call to `env.close()` in `verifyDeliveredAmount`. It's already doing almost exactly what I described.